lightning/src/ln/channel.rs

```diff
@@ -3136,13 +3138,13 @@ impl<ChanSigner: ChannelKeys> Channel<ChanSigner> {
 			// channel might have been used to route very small values (either by honest users or as DoS).
 			self.channel_value_satoshis * 9 / 10,
 
-			Channel::<ChanSigner>::get_our_max_htlc_value_in_flight_msat(self.channel_value_satoshis)
+			Channel::<ChanSigner>::get_holder_max_htlc_value_in_flight_msat(self.channel_value_satoshis)
 		);
 	}
 
 	/// Allowed in any state (including after shutdown)
-	pub fn get_their_htlc_minimum_msat(&self) -> u64 {
-		self.our_htlc_minimum_msat
+	pub fn get_counterparty_htlc_minimum_msat(&self) -> u64 {
+		self.holder_htlc_minimum_msat
```

Comment:
Not a new bug, but this looks very broken - I'm kinda sad we don't have test coverage for it.

Reply:
That's was the starting point of : https://github.com/rust-bitcoin/rust-lightning/pull/670, and we know how it ends up. Add a reference in #680

Reply:
Wait, I dont recall you mentioning this - I was under the impression 670 was NOTABUG, this is definitely a bug.